fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -297,6 +297,35 @@ class ApplyMetadataMutationsImpl {
 		}
 	}
 
+	void checkSetStorageServerToTeamIDKeyPrefix(MutationRef m) {
+		if (!m.param1.startsWith(storageServerToTeamIdKeyPrefix) || !SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS) {
+			return;
+		}
+		ASSERT_WE_THINK(tLogGroupCollection.isValid());
+		UID serverID = decodeStorageServerToTeamIdKey(m.param1);
+		ptxn::StorageServerStorageTeams teams(
+		    m.param2); // assuming it is full set, could it be true that it only contain incremental data?
```

Comment:
Remove the note? It has to be full set since it sets the entire value of the key.